sdks/python/apache_beam/transforms/combiners.py

```diff
@@ -88,7 +90,7 @@ def expand(self, pcoll):
 
 # TODO(laolu): This type signature is overly restrictive. This should be
 # more general.
-@with_input_types(Union[float, int])
+@with_input_types(Union[float, int, np.int64, np.float64])
```

Comment:
Note I had to make this typehint change so I could use CombinePerKey in one of my tests. Is this alright?

Reply:
Yes, this is fine.